package.json

```diff
@@ -16,13 +17,52 @@
     "pre-commit": "pre-commit run --all-files",
     "prepare": "npm run clean && npm run build",
     "prettier": "prettier --write .",
-    "rollup": "rollup -c",
+    "rollup": "wireit",
```

Comment:
You don't need to define every `wireit` command here (examples may be outdated on the `wireit` part as that is newish [CHANGELOG](https://github.com/google/wireit/blob/main/CHANGELOG.md#changed-1)), I think it would be better to only have a single `build` command.

Reply:
Ack